style/footer/_footer.sass

```diff
@@ -0,0 +1,41 @@
+.footer
+    @include displayflex(column nowrap,center,center)
+    padding: 30px 0px 30px 0px
+    background-color: $header-color
+
+    &__block
+        width: 100%
+
+.block
+    @include displayflex(row nowrap,center,space-around)
+    gap: 40px
+    margin-bottom: 30px
+
+    &__item
+        padding: 10px 0px 10px 0px
+    &__link
+        gap: 20px
+    &__link:hover
+        border-bottom: 1px solid $color-hover        
+
+.social-media__link
+    padding: 0px 20px 0 20px
+    font-size: 20px
+
+.fab.fa-facebook:hover,.fab.fa-instagram:hover,.fab.fa-twitter:hover
```

Comment:
one selector one line, so, also it can be simplified:

```
.fab.fa-facebook,
.fab.fa-instagram,
.fab.fa-twitter
    &:hover
         // styles
```